Sources/Identity+PublicAPI.swift

```diff
@@ -48,6 +48,37 @@ import Foundation
         }
     }
 
+    /// Returns the identifiers in URL query parameter format for consumption in hybrid mobile applications.
+    /// There is no leading & or ? punctuation as the caller is responsible for placing the variables in their resulting URL in the correct locations.
+    /// If an error occurs while retrieving the URL variables, the completion handler is called with a nil value and AEPError instance.
+    /// Otherwise, the encoded string is returned, for ex: `"adobe_mc=TS%3DTIMESTAMP_VALUE%7CMCMID%3DYOUR_ECID%7CMCORGID%3D9YOUR_EXPERIENCE_CLOUD_ID"`
+    /// The `adobe_mc` attribute is an URL encoded list that contains:
+    ///     - TS: a timestamp taken when the request was made
+    ///     - MCMID: Experience Cloud ID (ECID)
+    ///     - MCORGID: Experience Cloud Org ID
+    /// - Parameter completion: invoked with a value containing the identifiers in query parameter format or an AEPError if an unexpected error occurs or the request times out.
+    @objc(getUrlVariables:)
+    static func getUrlVariables(completion: @escaping (String?, Error?) -> Void) {
+        let event = Event(name: IdentityConstants.EventNames.REQUEST_IDENTITY_URL_VARIABLES,
+                          type: EventType.edgeIdentity,
+                          source: EventSource.requestIdentity,
+                          data: [IdentityConstants.EventDataKeys.URL_VARIABLES: true])
+
+        MobileCore.dispatch(event: event) { responseEvent in
+            guard let responseEvent = responseEvent else {
+                completion(nil, AEPError.callbackTimeout)
+                return
+            }
+
+            guard let urlVariables = responseEvent.data?[IdentityConstants.EventDataKeys.URL_VARIABLES] as? String else {
+                completion(nil, AEPError.unexpected)
+                return
+            }
+
+            completion(urlVariables, .none)
```

Comment:
I see the error is optional in the completion handler, why do we need to return AEPError.none here instead of nil?

Reply:
I was just following the other public APIs. We should update all to nil.

Reply:
The code is a bit misleading, this is **not** `AEPError.none`, it is `Swift.Optional.none` which is equivalent to `nil`.